fe/fe-core/src/main/java/com/starrocks/qe/ExecuteScriptExecutor.java

```diff
@@ -51,9 +51,6 @@ static ShowResultSet makeResultSet(String result) {
     }
 
     public static ShowResultSet execute(ExecuteScriptStmt stmt, ConnectContext ctx) throws UserException {
-        if (!"127.0.0.1".equals(ctx.getRemoteIP()) || Config.enable_remote_script) {
-            throw new UserException("script can only be executed on localhost");
-        }
         if (stmt.isFrontendScript()) {
             return executeFrontendScript(stmt, ctx);
         } else {
```

Comment:
just only remove limit for BE script? not FE(groovy script)?

Reply:
There is already a config param `enable_execute_script_on_frontend` for fe.